wgsl/index.bs

```diff
@@ -3581,6 +3600,15 @@ The `for(initializer; condition; continuing) { body }` statement is syntactic su
 * If `condition` is non-empty, it is checked at the beginning of the loop body and if unsatisfied then a [[#break-statement]] is executed.
 * If `continuing` is non-empty, it becomes a [[#continuing-statement]] at the end of the loop body.
 
+A declaration introduced in the body of the for loop is [=in scope=] until the
+end of the for loop's current iteration.
```

Comment:
Please adapt the wording from above.

Would this need to adapt the definition of the iteration?